### PR TITLE
Import music data

### PR DESCRIPTION
**Pull request by JosephCottingham**

### CODE REVIEW

**Comment by JosephCottingham** [comment]
Please do a squash and merge!

File: music_lib_search_tool/music_lib_search_tool/apps/music_collection_search/csv_cleaner.py [hunk 1]
```diff
@@ -170,10 +170,10 @@ def get_song_titles():
     genre_models_list = []
     instrument_models_list = []
     publisher_models_list = []
-    master_owner_models_list = [] 
+    master_owner_model = None
```
Comment (tristann3) [inline music_lib_search_tool/music_lib_search_tool/apps/music_collection_search/csv_cleaner.py:173]: update to master_owner_models_list

File: music_lib_search_tool/music_lib_search_tool/apps/music_collection_search/csv_cleaner.py [hunk 2]
```diff
@@ -170,10 +170,10 @@ def get_song_titles():
...
     mood_models_list = []
-    time_signature_model = [] 
-    mode_model = [] 
+    time_signature_model = None
```
Comment (tristann3) [inline music_lib_search_tool/music_lib_search_tool/apps/music_collection_search/csv_cleaner.py:175]: update to time_signature_models_list

File: music_lib_search_tool/music_lib_search_tool/apps/music_collection_search/csv_cleaner.py [hunk 3]
```diff
@@ -170,10 +170,10 @@ def get_song_titles():
...
+    mode_model = None
```
Comment (tristann3) [inline music_lib_search_tool/music_lib_search_tool/apps/music_collection_search/csv_cleaner.py:176]: update ro mode_models_list

**Review by tristann3 — CHANGES_REQUESTED** [review]
Looks good, I'd like to see some improvements on spacing between blocks for readability but I don't think that is neccesary right now. Please fix variable consitency then you'll get my thumbs up

**Review by GSPuniani — COMMENTED** [review]
Should we leave in is_exclusive as a nullable field for now instead of deleting it? I think we will use it later with the exclusive contact info.

**Comment by tristann3** [comment]
> Should we leave in is_exclusive as a nullable field for now instead of deleting it? I think we will use it later with the exclusive contact info.

I don't think we should do anything Niko doesn't explicitly ask of us. We are already pushing deadlines, lets get this up and running asap.

**Pull request merged**